app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,63 @@
+$font-color: #088da5;
+$black: #000000;
+
+body {
+
+  background-color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.